netflix/src/components/home/Today.tsx

```diff
@@ -0,0 +1,65 @@
+'use client';
+import { useEffect, useState } from 'react';
+import axios from '@/api/axios';
+import request from '@/api/request';
+import Image from 'next/image';
+import TopNImg from '@/public/icon/topN.svg';
+import PlusImg from '@/public/icon/plus.svg';
+import PlayBtn from '@/components/common/PlayBtn';
+import InfoImg from '@/public/icon/info.svg';
+
+interface Movie {
+  id: number;
+  poster_path: string;
+  title: string;
+}
+
+export default function Today() {
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [randomMovie, setRandomMovie] = useState<Movie | null>(null); // 랜덤 영화 상태 추가
+
+  useEffect(() => {
+    // 영화 정보를 불러오는 함수
+    async function fetchMovies() {
+      try {
+        const requests = await axios.get(request.fetchTopRated);
+        setMovies(requests.data.results);
+        const randomIndex = Math.floor(Math.random() * requests.data.results.length); // 랜덤 인덱스 생성
+        setRandomMovie(requests.data.results[randomIndex]); // 랜덤 영화 선택
+      } catch (error) {
+        console.error(error);
+      }
+    }
+
+    fetchMovies();
+  }, []);
+
+  return (
+    <div>
+      <div className="w-full h-[415px] bg-gradient-to-b from-black via-black to-black">
```

Comment:
`gradient` 속성이 적용이 안된 것 같아요! 
`via-transparent`로 중간에 투명도를 줘서 다시한번 확인을 해봐도 좋을 것 같아요! 👍 
```suggestion
      <div className="w-full h-[415px] bg-gradient-to-b from-black via-transparent to-black">
```